### ISSUE TITLE
There is a "tickPixelInterval", please add "minorTickPixelInterval"

### ISSUE THREAD

**Issue by awall** [highcharts/highcharts#13336]
# Description of the feature
I'm trying to get a major tick every 200 pixels, and a minor tick every 100 pixels (specifically, this is for a datetime axis in my case). The only way I've managed to do this is with a custom tick positioner that needs to know the graph's width and a lot of insane date math. The whole reason my team chose highcharts over a custom solution is that it was *supposed* to make these things easy - I'm finding that's not the case.

# Library related to the feature
Highcharts

You can vote for the feature by adding thumbs-up reaction for this post.

**Comment by sebastianbochan** [comment]
Hi @awall,
Thank you for your suggestion of feature request!

We should consider kind of minorTickPixelInterval like tickPixelInterval exists.

**Comment by pawelfus** [comment]
How about implementing `minorTickAmount` instead of `minorTickPixelInterval` ? Right now, `minorTickInterval = "auto"` sets five minor ticks in between. If that value could be configurable, then `pixel` is not needed right? In your case @awall instead of setting 100px you would just set `minorTickAmount = 1` to rendered one minor tick in the middle.

**PoC:**
Demo: https://jsfiddle.net/BlackLabel/oc70hqxm/

**Snippet:**
```js
(function(H) {
  H.wrap(
    H.Axis.prototype,
    'getMinorTickPositions',
    function(proceed) {
      const amount = this.options.minorTickAmount,
        interval = this.tickInterval,
        minorTicks = [];

      if (Highcharts.isNumber(amount) && this.tickPositions) {
        this.tickPositions.forEach(tick => {
          for (let i = 0; i < amount; i++) {
            minorTicks.push(
              tick + (interval * (i + 1) / (amount + 1))
            );
          }
        });
        return minorTicks;
      } else {
        return proceed.apply(this, Array.prototype.slice.call(arguments, 1));
      }
    }
  );
})(Highcharts);
```

Let us now if you find any problems with the plugin above.

**Comment by awall** [comment]
@pawelfus For my purposes, either option is fine. I agree with you that a configurable "minorTickInterval" is the most useful. This way, I can set my major axes on any basis I want (either # of ticks, pixels between ticks, custom tick function, etc...), and my minor ticks are just "x minor ticks per major tick", it handles many scenarios.

**Comment by TorsteinHonsi** [comment]
@pawelfus I agree to that approach. But we must be careful with the naming not to add to the confusion. Maybe something more verbose like `minorTicksPerTick` or `minorTicksPerMajorTick`?

**Comment by pawellysy** [comment]
unstale
